public/documents/fari-rpgs/charge-rpg.md

```diff
@@ -2334,6 +2334,300 @@ The original idea comes from [Ironsworn](https://ironswornrpg.com)'s vows. `Goal
+Create a new immediate opportunity - the GM exposes an opportunity and offers the players with a positive `supporting detail` _if_ they complete a mission to obtain it.
+
+> #### Example
+> 
+> The party is back at spacedock and has gathered enough progress to fill the progress clock. They collectively decide they would like to get the target off their back. They choose to improve the authority `supporting detail` “falsified spacecraft registration” (-), and rewrite it as “overhauled ship hides stolen past” (0). They haven't completely gotten rid of the mysterious origin of the ship, but it won’t put them in such disadvantageous `positions` any more.
+
+## Other Uses for Key Concepts and Supporting Details
+
+> #### Designer Note
+> 
+> Certain games may pick and choose between the following options - while they do not add a large amount of complexity, using the full set of rules grants the players many additional options to see them through tough spots. 
+
+### Supporting Details Grant Conditional Position
+
+When dealing with a situation that directly relates to a `supporting detail`, the `disposition` of the supporting detail (positive or negative) may be used to help determine the `position` of the roll. 
+
+### Key Concepts Grant Conditional Position
+
+When dealing with a situation that might benefit from a positive `key concept`, the `disposition` of the `key concept` may be used to help determine the `position` of the roll. Since `key concepts` are generic to category descriptors, this gives a much broader benefit than individual `supporting details`. 
+
+The same works in reverse - allowing a `key concept` to become broadly negative can make the lives of the party more desperate.
+
+### Improved Effect
+
+When dealing with a situation for which a `supporting detail` or `key concept` could be a useful advantage, players may opt for +1 effect rather than an improved position. Which option is being used must be decided before the roll. 
+
+### Losing Progress via Add Tension
+
+Progress toward improving details can be spent as momentum. Taking it a step further - if it makes sense, the players might suggest downgrading the `disposition` of a supporting detail as a consequence to `add tension` if they don’t have progress to spare. 
+
+## Dials
+
+These rules can be adapted in a few different ways. Below are some ways to tweak them to better fit how the play group wants to use them. 
+
+### No Clocks
+
+Rather than track progress and danger on abstract clocks, the play group might decide to update the `party sheet` whenever it makes sense narratively. Rather than keep track of additional clocks, the players can decide to update details 
```

Comment:
```suggestion
Rather than track progress and danger on abstract clocks, the party might decide to update the `party sheet` whenever it makes sense narratively. Rather than keep track of additional clocks, the players can decide to update details 
```

You alrady define the term, and the repetition here can serve as an emphasis to the extra